@planetarium/account-aws-kms/src/AwsKmsAccount.ts

```diff
@@ -0,0 +1,36 @@
+import { AwsKmsKeyId } from "./AwsKmsKeyId.js";
+import { KMSClient, SignCommand } from "@aws-sdk/client-kms";
+import { Signature as NobleSignature } from "@noble/secp256k1";
+import {
+  type Account,
+  type Message,
+  PublicKey,
+  Signature,
+} from "@planetarium/account";
+
+export class AwsKmsAccount implements Account {
+  readonly #client: KMSClient;
+
+  readonly keyId: AwsKmsKeyId;
+  readonly publicKey: PublicKey;
+
+  constructor(keyId: AwsKmsKeyId, publicKey: PublicKey, client: KMSClient) {
+    this.keyId = keyId;
+    this.publicKey = publicKey;
+    this.#client = client;
+  }
+
+  async sign(message: Message): Promise<Signature> {
+    const cmd = new SignCommand({
+      KeyId: this.keyId,
+      Message: message,
+      SigningAlgorithm: "ECDSA_SHA_256",
+    });
+    const response = await this.#client.send(cmd);
+    if (response.Signature == null) throw new Error("Failed to sign message");
+    const sig = NobleSignature.fromDER(response.Signature).normalizeS();
```

Comment:
Erm... Some future- no. present-proof note here,
https://github.com/paulmillr/noble-secp256k1/pull/92
Juuuuuust recently (about 4 days ago) @noble/secp256k1 released new v2.0,
and this effectively dropped all DER related methods, including `fromDER()`, `toDERRawBytes()`. `toDERHex()`.

We may need some action about this part, I guess.

Also, High-S detection / S normalization now seems to be integrated as part of `prepSig()` 
https://github.com/paulmillr/noble-secp256k1/blob/ec7b5731808538c7cc32cba3234822a003958345/index.ts#L249
yous might want to look into.

Reply:
noble secp 2 has not been released yet

Reply:
Oh, I just found out that after checking npm too, haha. thanks for noticing on here too!